<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainKey51</class>
 <widget class="QDialog" name="MainKey51">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>221</width>
     <height>258</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>终端参数</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>79</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>交易模式</string>
     </property>
     <widget class="QRadioButton" name="radioButton_14">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>14</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>非一体机</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_15">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>35</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>一体机</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>144</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>定额设置</string>
     </property>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>18</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>是</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>54</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>36</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>交易金额/分</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>57</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>37</y>
       <width>57</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>流水号</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>78</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>打印机</string>
     </property>
     <widget class="QRadioButton" name="radioButton_16">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>10</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>热敏</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_17">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>31</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>针打</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_18">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>50</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>套打</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>153</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>金额设置</string>
     </property>
     <widget class="QRadioButton" name="radioButton_19">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>13</y>
        <width>98</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>以分为单位</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_20">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>31</y>
        <width>98</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>以角为单位</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_21">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>49</y>
        <width>98</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>以元为单位</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>37</y>
       <width>57</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>终端号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>57</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>000000</string>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>57</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>商户号</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>通讯参数</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>通讯模式</string>
     </property>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>22</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>拨号</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>40</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>网络</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_11">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>60</y>
        <width>57</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>GPRS</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_12">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>80</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>串口</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_13">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>100</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>CDMA</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>211</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>拨号参数</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>36</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>主拨号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>55</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>备份</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>71</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>36</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>16</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>分机号</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>网络参数</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>36</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>19</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>主机IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>56</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>端口号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>73</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>92</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>APN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>5.1 系统参数设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>退  出</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>Title</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
